tools/proteomiqon_peptidespectrummatching/proteomiqon_peptidespectrummatching.xml

```diff
@@ -0,0 +1,196 @@
+<tool id="proteomiqon_peptidespectrummatching" name="ProteomIQon PeptideSpectrumMatching" version="@VERSION@" profile="20.05">
+    <description>
+        iterates across all MS/MS scans in an MS run, determines precursor charge states and possible peptide spectrum matches.
+    </description>
+    <macros>
+        <token name="@VERSION@">0.0.7</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">proteomiqon-peptidespectrummatching</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        #import re
+        #set basename = $re.sub(r'[^\w ,.\-+]','_',$instrumentOutput.element_identifier)
+        #if $outputParamfile:
+            cat '$paramfile' >> '$out_paramfile' &&
+        #end if
+        ln -s $instrumentOutput '${basename}.mzlite' &&
+        ln -s $out_psm '${basename}.psm' &&
+        proteomiqon-peptidespectrummatching -i './${basename}.mzlite' -d '$peptideDB' -p '$paramfile' -o ./
+    ]]>
+    </command>
+    <configfiles>
+        <configfile name="paramfile">
+        <![CDATA[
+        {
+            "ChargeStateDeterminationParams": {
+                "ExpectedMinimalCharge": ${ChargeStateDeterminationParams.ExpectedMinimalCharge},
+                "ExpectedMaximumCharge": ${ChargeStateDeterminationParams.ExpectedMaximumCharge},
+                "Width": ${ChargeStateDeterminationParams.Width},
+                "MinIntensity": ${ChargeStateDeterminationParams.MinIntensity},
+                "DeltaMinIntensity": ${ChargeStateDeterminationParams.DeltaMinIntensity},
+                "NrOfRndSpectra": ${ChargeStateDeterminationParams.NrOfRndSpectra}
+            },
+            "LookUpPPM": ${LookUpPPM},
+            "nTerminalSeries": { "Case": "${nTerminalSeries}" },
+            "cTerminalSeries": { "Case": "${cTerminalSeries}" },
+            "Andromeda": {
+                "PMinPMax": {
+                "Item1": ${Andromeda.PMin},
+                "Item2": ${Andromeda.PMax}
+                },
+                "MatchingIonTolerancePPM": ${Andromeda.MatchingIonTolerancePPM}
+            }
+        }
+        ]]>
+        </configfile>
+    </configfiles>
+    <inputs>
+    <param name="instrumentOutput" type="data" format="sqlite" label="Instrument output" help="Specify mass spectrometry data you want to analyze."/>
+    <param name="peptideDB" type="data" format="sqlite" label="Peptide database" help="Specify the peptide data base."/>
+    <section name="ChargeStateDeterminationParams" title="Charge state determination parameters" >
+        <param name="ExpectedMinimalCharge" type="integer" value="2" label="Expected minimal charge" help="Specify the minimum peptide ion charge state to consider."/>
```

Comment:
would it make sense to add min= max= to a few of the floats/integers here?

Reply:
Added